examples_tests/13.BRDF_Explorer/CMakeLists.txt

```diff
@@ -4,9 +4,16 @@ if(NOT RES)
 	message(FATAL_ERROR "common.cmake not found. Should be in {repo_root}/cmake directory")
 endif()
 
+add_compile_definitions(CEGUI_STATIC CEGUI_BUILD_STATIC_FACTORY_MODULE)
+
+set(BRDF_EXPLORER_LIB
```

Comment:
BRDF Explorer will be a lib?

Reply:
BRDF_EXPLORER_LIB variable stores the library list (which the app needs to link).